src/SharedStreamManager.mo

```diff
@@ -57,6 +58,25 @@ module {
     /// principals and id-s of registered cross-canister stream sources
     public func canisterStreams() : Vec.Vector<(Principal, ?Nat)> = Vec.fromIter(List.toIter(sourceCanistersStreamMap));
 
+    /// principals of cross-canister stream surces with the priority. The priority value tells the caller
+    /// with what probability he should chose that canister for their needs. In future this value will be used for
+    /// load balancing calls, for now it returns either 0 or 1. Zero value means that stream is closed it the canister should not be used
+    public func prioritySourceCanisters() : Vec.Vector<(Principal, Nat)> = Vec.fromIter(
```

Comment:
Why return a vector, not an array? Shouldn't it be an immutable array?

Reply:
Done